docs/designs/keras_callback.md

```diff
@@ -0,0 +1,208 @@
+# Design to Support Keras Callback
+
+This document describes the design for supporting callback to customize the behavior of model during training, evaluation and inference in ElasticDL.
+
+## Motivation
+
+In deep learning, we generally need to customize the behavior of model during training, evaluation or
```

Comment:
What's the meaning of `changing the model`?

Reply:
For example, users may check the model weights and change it. But, we don't have a use case of changing the model now.

Reply:
Can you please share a scenario of `check the model weights and change it` with the native keras?

Reply:
Emmm, i don't have a scenario. But i think changing the learning rate is changing the model.